crypto/README.rst

```diff
@@ -11,8 +11,8 @@ Crypto Libraries
    doc/nrf_cc310_platform
    doc/nrf_cc310_mbedcrypto
    doc/nrf_oberon
-   nrf_cc310_platform/CHANGELOG.rst
-   nrf_cc310_bl/CHANGELOG
-   nrf_cc310_mbedcrypto/CHANGELOG
-   nrf_oberon/CHANGELOG
+   doc/CHANGELOG_cc310_bl
```

Comment:
Does it matter that the previous link specified the RST file extension and the new one doesn't?

Reply:
I don't think so as all the other files don't have it I aligned it with the rest.